storage/src/blockchain_db_adapter.cpp

```diff
@@ -111,7 +110,7 @@ int KeyManipulator::composedKeyComparison(const uint8_t* _a_data, size_t _a_leng
       return (aId > bId) ? 1 : (bId > aId) ? -1 : 0;
     }
     default:
-      LOG_ERROR(logger(), "KeyManipulator::composedKeyComparison called with invalid key type: " << (char) aType);
+      LOG_ERROR(logger(), "invalid key type: " << (char) aType);
```

Comment:
Why remove the context around the error?

Reply:
See my logger comment above